nengo_gui/components/pointer.py

```diff
@@ -16,17 +15,34 @@ def __init__(self, obj, **kwargs):
         # the semantic pointer value as set by the user in the GUI
         # a value of 'None' means do not override
         self.override_target = None
-        self.vocab_in = obj.inputs[self.target][1]
+        loop_in_whitelist = [spa.Buffer, spa.Memory, spa.State, spa.AssociativeMemory]
```

Comment:
> That's how self.target is defined in SpaPlot.

No it's not.  The line in SpaPlot says ```self.target = kwargs.get('args', 'default')``` which means that it *defaults* to ```'default'```, but if someone passes it in to the constructor something like ```Pointer(target='A')``` then it'll display that output instead.  Admittedly, we don't currently use that argument, but the fact that it's there and we have specific code in the constructor to handle setting it means that we shouldn't assume that it's always set to ```'default'```.